packages/warehouses/src/warehouseClients/DatabricksWarehouseClient.ts

```diff
@@ -189,54 +215,49 @@ export class DatabricksWarehouseClient implements WarehouseClient {
             table: string;
         }[],
     ) {
-        let pool: odbc.Pool;
-        let results: Result<SparkSchemaResult>[];
-        try {
-            pool = await odbc.pool(this.connectionString);
-        } catch (e) {
-            throw new WarehouseConnectionError(e.message);
-        }
+        const { session, close } = await this.getSession();
+        let results: SchemaResult[][];
+
         try {
             const promises = requests.map(async (request) => {
-                let connection: odbc.Connection;
+                let query: IOperation | null = null;
                 try {
-                    connection = await pool.connect();
-                } catch (e) {
-                    throw new WarehouseConnectionError(e.message);
-                }
-                try {
-                    const columns = (await connection.columns(
-                        // @ts-ignore
-                        'SPARK', // This is always SPARK
-                        request.schema,
-                        request.table,
-                        '',
-                    )) as Result<SparkSchemaResult>;
-                    return columns;
+                    query = await session.getColumns({
+                        catalogName: request.database,
+                        schemaName: request.schema,
+                        tableName: request.table,
+                    });
+
+                    const result = (await query.fetchAll()) as SchemaResult[];
+
+                    return result;
                 } catch (e) {
                     throw new WarehouseQueryError(e.message);
                 } finally {
-                    await connection.close();
+                    if (query) query.close();
                 }
             });
             results = await Promise.all(promises);
         } finally {
-            await pool.close();
+            await close();
         }
+
         return results.reduce<WarehouseCatalog>(
             (acc, result, index) => {
                 const columns = Object.fromEntries<DimensionType>(
                     result.map((col) => [
                         col.COLUMN_NAME,
-                        mapFieldType(col.TYPE_NAME),
+                        mapFieldType(col.DATA_TYPE),
                     ]),
                 );
                 const { schema, table } = requests[index];
-                acc.SPARK[schema] = acc.SPARK[schema] || {};
-                acc.SPARK[schema][table] = columns;
+
+                acc[this.catalog][schema] = acc[this.catalog][schema] || {};
```

Comment:
I think this is going to cause problems in `attachTypesToModels` in `packages/common/src/compiler/translator.ts` and maybe other places. 
The code expects the catalog keys to be the `database` value in the dbt models.

Reply:
I think it is being mapped correctly

I tested:
- creating project
- refreshing dbt project
- running sql query
- using explore to generate sql query

sql runner:
![CleanShot 2022-10-10 at 13 55 05@2x](https://user-images.githubusercontent.com/962095/194840862-730fb9b9-fd9e-4d17-814f-649b18ca34a9.png)

explore:
![CleanShot 2022-10-10 at 13 57 34@2x](https://user-images.githubusercontent.com/962095/194841153-635f4543-652f-47ae-aafd-015de954471f.png)